### PR TITLE
Switch from Context to Extension

### PR DESCRIPTION
**Pull request by fauxalgore**
Fulfills https://github.com/fauxalgore/behat-say/issues/2
Upon merging this PR, rename the repo: https://github.com/fauxalgore/behat-say/issues/5

### CODE REVIEW

**Comment by stevector** [comment]
@fauxalgore Can you put CircleCI in place before merging this PR? https://github.com/fauxalgore/behat-say/issues/3

It would be a good idea to have at least code sniffing in place before new PHP files are added.

**Comment by fauxalgore** [comment]
> It would be a good idea to have at least code sniffing in place before new PHP files are added.

That is a fair point. However I do want to get the repo renamed before enabling CircleCI as I am not certain Circle follows renaming well.

File: src/ServiceContainer/BehatSayExtension.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<?php
+
+
+namespace FauxAlGore\BehatSayExtension\ServiceContainer;
+
+use Behat\Behat\Context\ServiceContainer\ContextExtension;
```
Comment (stevector) [inline src/ServiceContainer/BehatSayExtension.php]: @fauxalgore, please remove unused classes from this section.

File: src/BehatSaySubscriber.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    public static function getSubscribedEvents() {
+        return array(
+            BeforeStepTested::BEFORE   => 'BeforeStep',
+            AfterStepTested::AFTER  => 'afterStep',
+
+        );
+    }
+}
```
Comment (stevector) [inline src/BehatSaySubscriber.php]: linebreak.

File: src/BehatSaySubscriber.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+namespace FauxAlGore\BehatSayExtension;
+
+use Behat\Behat\EventDispatcher\Event\BeforeStepTested;
+use Behat\Behat\EventDispatcher\Event\AfterStepTested;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class BehatSaySubscriber implements EventSubscriberInterface
+{
+
+    public function BeforeStep(BeforeStepTested $scope) {
```
Comment (stevector) [inline src/BehatSaySubscriber.php]: Is `scope` the right variable name?

**Pull request merged**